packages/tools/src/store/SynchronizerManager/Synchronizer.ts

```diff
@@ -66,6 +66,13 @@ class Synchronizer {
     this._viewportOptions[viewportId] = options;
   }
 
+  /**
+   * Sets a synchronizer enabled
+   */
+  public setEnabled(enabled: boolean) {
```

Comment:
can we add a setDisabled too?

Reply:
That isn't normally how one does it - just call setEnabled(false)